pyscriptjs/src/pyodide.ts

```diff
@@ -97,6 +97,32 @@ export class PyodideRuntime extends Runtime {
         }
     }
 
+    /**
+     *
+     * @param path : the path in the filesystem
+     * @param fetch_path : the path to be fetched
+     *
+     * For a given resource with `fetch_path` as `http://dummy.com/hi.py`
+     * and the location of `path` in the FS to be `a/b/c/foo.py`
+     *
+     * Iteratively analyses the paths:
+     *  - `a`
+     *  - `a/b`
+     *  - `a/b/c`
+     *
+     * The analysis returns if the path exists and if it's parent directory exists
+     * If the path itself doesn't exist, we create it
+     *
+     * Due to the manner in which we proceed, the parent will ALWAYS exist.
```

Comment:
Similarly to the comment above, I think this over complicates the message.... As a reader, I think I'd be easier for me to understand if we just said something like: "Given a nested `path` such as `a/b/c`, it'll recursively check if each part of the path exists and create it if not."

Reply:
I wanted to give a detailed example, but I can make it shorter of course 😅

Reply:
Done, although the approach is `iterative` here and not recursive.